crates/store/re_grpc_client/src/lib.rs

```diff
@@ -376,3 +387,104 @@ async fn stream_catalog_async(
 
     Ok(())
 }
+
+// Craft a blueprint from relevant chunks and activate it
+fn activate_catalog_blueprint(
+    tx: &re_smart_channel::Sender<LogMsg>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let blueprint_store_id =
+        StoreId::from_string(StoreKind::Blueprint, "catalog_blueprint".to_owned());
+    let blueprint_store_info = StoreInfo {
+        application_id: ApplicationId::from("redap_catalog"),
+        store_id: blueprint_store_id.clone(),
+        cloned_from: None,
+        is_official_example: false,
+        started: Time::now(),
+        store_source: StoreSource::Unknown,
+        store_version: None,
+    };
+
+    if tx
+        .send(LogMsg::SetStoreInfo(SetStoreInfo {
+            row_id: *re_chunk::RowId::new(),
+            info: blueprint_store_info,
+        }))
+        .is_err()
+    {
+        re_log::debug!("Receiver disconnected");
+        return Ok(());
+    }
+
+    let timepoint = [(Timeline::new_sequence("blueprint"), 1)];
```

Comment:
`re_viewer_context::blueprint_timeline()`

Reply:
I'm actually not sure it's worth bringing in ``re_viewer_context`` to grpc client just for this one.

Reply:
ah, right, I see suggestions below are also from this crate...

Reply:
and adding it would create a dependency cycle :/

Reply:
Yeah I was expecting that dependency cycle. I dont think we should do anything about it. My suggestion is to simply add a bunch of todos, and maybe open an issue to document this.

Reply:
Had a brief huddle with @abey79 and he explained a bit the dependencies and what I'm hitting here makes sense and we don't want grpc client to depend on ``re_viewer_context``. When rust blueprint API is properly introduced, we might depend on that crate (something like ``re_blueprint``), but arguably by then catalog view blueprint might simply move to Data Platform and live along other customer recording's blueprints. 

For now we'll go with what we have here and I'll add some TODOs in the code.